var/spack/repos/builtin/packages/cube-blade/package.py

```diff
@@ -10,14 +10,15 @@ class CubeBlade(AutotoolsPackage):
     """Simple OTF2 trace explorer"""
 
     homepage = "https://www.scalasca.org/software/cube-4.x/download.html"
-    url = "https://apps.fz-juelich.de/scalasca/releases/cube/4.5/dist/blade-0.2.tar.gz"
+    url = "https://apps.fz-juelich.de/scalasca/releases/cube/4.8/dist/blade-0.5.tar.gz"
```

Comment:
Cannot confirm the version sha256 for this version without a correction to the `url`:

```suggestion
    url = "https://apps.fz-juelich.de/scalasca/releases/cube/4.8/dist/blade-v0.5.tar.gz"
```